src/main/java/com/marginallyclever/convenience/ToggleButtonIcon.java

```diff
@@ -52,4 +50,16 @@ private static ImageIcon createImageIcon(String path) {
             return null;
         }
     }
+
+    public static class Item {
```

Comment:
Ambiguous name is ambiguous.

Reply:
I asked myself the same but as you can only instanciate it by specifying `new ToggleButtonIcon.Item()` I think it will be OK